lib/active_fedora/loadable_from_json.rb

```diff
@@ -105,6 +105,30 @@ def reflection(predicate)
       end
     end
 
+    # Adapt attributes read from Solr to possible minor changes in data model
+    # since the attributes were saved.
+    # @param attrs [Hash] attributes read from Solr
+    # @return [Hash] the adapted attributes
+    def adapt_attributes(attrs)
+      new_attributes = {}
+      self.class.attribute_names.each do |a|
```

Comment:
Can you set this `a` variable to something more descriptive? How about `attribute_name`? Also consider using `each_with_object({}) do |attribute_name, new_attributes|` here. Then you can remove the previous line and the last line of the method.